tutorials/get-started-bitnami-django.md

```diff
@@ -0,0 +1,202 @@
+
+## Install Bitnami Django on a Google Compute Engine instance
+
+In this section, you will deploy Bitnami Django on a Google Compute Engine instance.
+
+1. From the Google Cloud Platform menu, select the [Cloud Launcher](https://console.cloud.google.com/launcher).
+1. Search for "django certified by bitnami" and select the resulting `Django Certified by Bitnami` template.
+1. Review the information and costs. Click `Launch on Compute Engine` to proceed.
+1. Review the default zone, machine type, boot disk size and other parameters and modify as needed. Ensure that the `Allow HTTP traffic` and `Allow HTTPS traffic` boxes are checked in the firewall configuration. Click `Deploy` to proceed with the deployment.
+
+The Cloud Launcher deploys Bitnami Django on a new Google Compute Engine instance. You will be able to monitor the progress of the deployment from the [Deployment Manager](https://console.cloud.google.com/dm/deployments). Once deployed, note the public IP address of the instance and the password for the MySQL and PostgreSQL databases.
+
+## Create a "hello world" Django application
+
+In this section, you will log in to the deployed instance and create a simple Django application.
+
+1. From the [Deployment Manager](https://console.cloud.google.com/dm/deployments), click the `SSH` button to log in to the instance over SSH.
+1. Once logged in, switch to the `bitnami` user account:
+
+        sudo su - bitnami
+
+1. Navigate to the default Django projects folder:
+
+        cd /opt/bitnami/apps/django/django_projects
+
+1. Create a new project using the command below:
+
+        django-admin.py startproject myproject
+
+1. Create a new application skeleton within the project:
+
+        cd myproject
+        python3 manage.py startapp helloworld
+
+1. Edit the `/opt/bitnami/apps/django/django_projects/myproject/helloworld/views.py` file and add this content:
+
+        from django.http import HttpResponse
+
+        def index(request):
+          return HttpResponse("Hello world!")
+
+1. Create the `/opt/bitnami/apps/django/django_projects/myproject/helloworld/urls.py` file and add these lines to it:
+
+        from django.conf.urls import url
+        from . import views
+
+        urlpatterns = [
+            url(r'^$', views.index, name='index'),
+        ]
+
+1. Edit the `/opt/bitnami/apps/django/django_projects/myproject/myproject/urls.py` file and modify it to look like this:
+
+        from django.conf.urls import url
+        from django.urls import include
+        
+        urlpatterns = [
+            url(r'^helloworld/', include('helloworld.urls')),
+        ]
+
+## Serve the application with Apache
+
+Bitnami Django includes a pre-configured instance of the Apache Web server. In this section, you will configure Apache to serve the application on the standard Web server port 80.
```

Comment:
suggested edit: "Configure Apache to serve the application on the standard Web server port 80:"